llvm/lib/Transforms/Instrumentation/MemorySanitizer.cpp

```diff
@@ -5623,7 +5620,6 @@ struct MemorySanitizerVisitor : public InstVisitor<MemorySanitizerVisitor> {
                             getOriginPtrForArgument(IRB, ArgOffset));
           }
         }
-        (void)Store;
```

Comment:
In release builds:
```
llvm/lib/Transforms/Instrumentation/MemorySanitizer.cpp:5566:16: error: variable 'Store' set but not used [-Werror,-Wunused-but-set-variable]
 5566 |         Value *Store = nullptr;
```

Reply:
oops, meant to send this as draft. thanks.

Reply:
AI would have automatically caught this and improved your productivity by 3%